dracut/30ignition/ignition-diskful-udev.service

```diff
@@ -0,0 +1,25 @@
+[Unit]
+Description=Ignition Diskful UDEV Trigger
+Documentation=https://github.com/coreos/ignition
+
+Before=ignition-diskful.target
+Before=ignition-diskful-subsequent.target
+Before=ignition-subsequent.target
+
+After=ignition-setup-user.service
+After=ignition-setup-base.service
+
+# Wait for multipathd
+Wants=multipathd.service
```

Comment:
One thing related to this...it'd be nice if we only started `multipathd.service` if it was actually in use; today we start it unconditionally in the initramfs and it just spews an error and then nothing happens.  (And the same with iscsi)

IOW I think this should only be started if there's a `dm.multipath=1` karg or something like that.

Reply:
Ah but I guess this whole service is only started if something like multipath is in use?

If that's the case, bikeshed naming: `ignition-diskful-complex-blockdev.service` ?

Reply:
I think the latest push is a bit better in this regard.